### ISSUE TITLE
Stuck forever at "Solution comparison is being refreshed, page will reload when done..."

### ISSUE THREAD

**Issue by Tom-H-L** [codewars/codewars.com#2983]
**Describe the bug**
Since over 48 hours the comparison with a new ally who also happens to be a newly registered user has stuck at this message and no comparisons are displayed

**To Reproduce**
Problem remains

**Expected behavior**
See the comparisons

**Screenshots**
can be created if deemed necessary

**Browser**
Firefox

**Comment by Blind4Basics** [comment]
Hello,

You're talking about the red head-to-head comparison, that's it?
A link to your CW account and the one of your friend might help here.

**Comment by Tom-H-L** [comment]
> Hello,

Hi, thanks for looking into this!

> You're talking about the red head-to-head comparison, that's it? 

Yes!

> A link to your CW account and the one of your friend might help here.

Sure, here you go:
[Me](https://www.codewars.com/users/Tom%20H.%20Lautenbacher)
[My ally](https://www.codewars.com/users/ocubist)

I also attached a screenshot for whatever that might be worth: ![image](https://user-images.githubusercontent.com/76924808/230782374-48845db1-fa29-46fd-ac5d-d4f7bf41597c.png)

**Comment by Blind4Basics** [comment]
Ok, I tried on my side: I can display both h2h comparisons without problems. 
Could you plz try in a private window? If it works there, it's most likely one of your browser extension that is the cause of the problem.

**Comment by Tom-H-L** [comment]
> 



> Could you plz try in a private window? If it works there, it's most likely one of your browser extension that is the cause of the problem.

That's interesting. I just tried it out with a private window of Firefox (with all plugins disabled) plus with Chromium and in both cases the problem remains on my side.

**Comment by Blind4Basics** [comment]
Then the admins will have to look into that...

**Comment by kazk** [comment]
You only train in Ruby, and ocubist doesn't, so there's nothing to compare.

The message should be updated.

**Comment by Tom-H-L** [comment]
> You only train in Ruby, and ocubist doesn't, so there's nothing to compare.

Oh I see! 
Well but that does not make sense in terms of a great solution, no? I mean: Why shouldn't I be able to see the solution that my ally came up with, even if it is in another language? Why is codewars deliberately hiding this from me, in this case, where it could very well just show me his code? This would even promote the various languages, as I could compare my language to the language of my ally and learn about a new language and the other way around!

So I would turn this into a feature request to have the other solutions of the same exercises shown, even if the languages are different! At last it should be an option to be able to see it if not even the default.

**Comment by hobovsky** [comment]
Solutions in other languages are hidden, because you get a reward in a form of a language rank for every language you solve. If you solved a kata in JS and then saw Ruby solutions, this would make the potential reward for Ruby pointless.

**Comment by Tom-H-L** [comment]
> Solutions in other languages are hidden, because you get a reward in a form of a language rank for every language you solve. If you solved a kata in JS and then saw Ruby solutions, this would make the potential reward for Ruby pointless.

Indeed, makes sense! So then it really just boils down to what kazk proposed: Just taking care of the misleading message that says since days "Solution comparison is being refreshed, page will reload when done..." and made me think that I can expect results any soon. Adding a message that says "No katas trained in same language" or something like that would add additional information to make clear why nothing is being displayed.
